src/test/spec/retryable_writes/test_file.rs

```diff
@@ -8,46 +8,46 @@ use crate::{
 
 #[derive(Debug, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct TestFile {
-    pub run_on: Option<Vec<RunOn>>,
-    pub data: Vec<Document>,
-    pub tests: Vec<TestCase>,
+pub(crate) struct TestFile {
```

Comment:
these being `pub` actually ran into some issues when trying to use `pub(crate)` stuff in the driver, and this happens because `cargo` can't really tell the difference between regular `pub` and `cfg(test)`'d `pub`. Since the tests are always crate private, I just converted everything to `pub(crate)`. I started with only a few things, but it has kind of a viral effect, leading to a lot of things needing to change, including stuff outside of the unified runner like this.